### PR TITLE
Fix coverage percentage

### PR DESCRIPTION
**Pull request by fynsta**
Because it only includes files that are actually loaded. This adds a useful shell script for generating a file that just randomly imports all other files... That file will also be run in the pipeline.

More info on the flutter issue: https://github.com/flutter/flutter/issues/27997

### CODE REVIEW

File: scripts/coverage_helper.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+file=test/coverage_helper_test.dart
+printf "// Helper file to make coverage work for all dart files\n" > $file
+printf "// **************************************************************************\n" >> $file
+printf "// Because of this: https://github.com/flutter/flutter/issues/27997#issue-410722816\n" >> $file
+printf "// DO NOT EDIT THIS FILE USE: ./scripts/file_finder.sh\n" >> $file
+printf "// **************************************************************************\n" >> $file
+printf "\n" >> $file
+printf "// ignore_for_file: unused_import\n" >> $file
+find lib -type f \( -iname "*.dart" ! -iname "*.g.dart" ! -iname "*.freezed.dart" ! -iname "generated_plugin_registrant.dart" \) | cut -c4- | awk -v package="$1" '{printf "import '\''package:motis_mitfahr_app%s'\'';\n", $1}' >> $file
```
Comment (Eddie-42) [inline scripts/coverage_helper.sh:11]: Yes I can definitely understand and am qualified to review this

Comment (fynsta) [reply scripts/coverage_helper.sh:11]: Basically it finds all files that end with ".dart", but not ".g.dart" etc. And then it creates an import line for each of those file names.

You can just run the script and see the file that it results in.

**Review by Showcas — APPROVED** [review]

**Review by Eddie-42 — APPROVED** [review]

**Comment by Eddie-42** [comment]
Does this also impact the comments by CodeCov/How does that work?

**Comment by fynsta** [comment]
> Does this also impact the comments by CodeCov/How does that work?

yes it does. Codecov gets sent the results of the test run in the pipeline. Since I put that shell script in the pipeline immediately before that step, this will be applied.

**Pull request merged**